lib/es/importDataFromFile.js

```diff
@@ -40,20 +41,35 @@ async function getPreviousIndex(alias) {
   return indexCurrent || alias;
 }
 
-async function importDataFromFile(filename, index, threshold) {
+async function importDataFromFile(filename, indexAlias, threshold) {
   try {
-    const latestIndex = `${index}_${getDateStamp()}`;
+    // if running locally, wait for ES to start
+    if (config.env === 'development') {
+      await esClient.waitForEsToStart();
+    }
+
+    const latestIndex = `${indexAlias}_${getDateStamp()}`;
     const data = transformData(loadJson(filename));
     const result = await esClient.createIndex(latestIndex, data);
 
-    const previousIndex = await getPreviousIndex(index);
+    const previousIndex = await getPreviousIndex(indexAlias);
     await validateData(result.count, previousIndex, threshold);
 
-    log.info(`deleting previous data in index '${previousIndex}'`);
-    await esClient.delete(previousIndex);
-
-    log.info(`creating alias to from ${latestIndex} to index '${index}'`);
-    await esClient.createAlias(latestIndex, index);
+    // there will be no previous alias if the initial index was the alias name.
+    // This will only happen the first time the obsolete 'db-profiles-elastic' image is used.
+    // in this case, delete must happen before the index is created to avoid a naming conflict.
+    // in normal operation the aliases will be swapped, then the old index deleted.
+    if (previousIndex === indexAlias) {
+      log.info(`deleting previous data in index '${previousIndex}'`);
```

Comment:
Does this log statement belong within the `esClient.delete` function? Saves having to remember to log it before calling it.